### ISSUE TITLE
AMaZE a bit less sharp with the new multipliers (at least in some cases)

### ISSUE THREAD

**Issue by ilia3101** [ilia3101/MLV-App#146]
Two screenshots, could not get image saving to work.

![screenshot from 2019-02-03 23-25-33](https://user-images.githubusercontent.com/23642861/52309178-3b1bf800-2997-11e9-8492-5c668727443c.png)
![screenshot from 2019-02-03 23-25-28](https://user-images.githubusercontent.com/23642861/52309179-3b1bf800-2997-11e9-9e8c-9b8710b24d25.png)

Open in browser tabs, switch between to pixel peep.

Just making this issue so I don't forget.

**Comment by masc4ii** [comment]
Interesting. I agree it was a tiny little bit sharper before in your example. Can't find that with my own examples. For me just aliasing and artifacts are a tiny little bit better now (mostly cyan and magenta).

**Comment by masc4ii** [comment]
Have you had a look, what happens, if you comment out the WB converting completely? Is it still sharper then?

**Comment by ilia3101** [comment]
> Have you had a look, what happens, if you comment out the WB converting completely? Is it still sharper then?

Doesn't this just make there be loads of moire like in the very early days of MLV App? Or I misinterpreted what that means?

**Comment by masc4ii** [comment]
No, you're right. But I don't know what should blur the picture. So if the new algorithm blurs, the old should blur too. Maybe the sharpest is untouched... nothing more was my question and idea.

**Comment by ilia3101** [comment]
I want to add a "Debayer white balance" option - a slider inside the raw corrections where you can set the temperature, maybe tweaking it would help for these cases where it's less sharp.

**Comment by bouncyball-git** [comment]
May be that Debayer WB also could be done with picker? Should be more accurate.

**Comment by ilia3101** [comment]
would be a good option to have as well

**Comment by masc4ii** [comment]
Do don't expect big differences - just some more or less cyan/green edge artifacts. So I don't think for now a picker is needed. In the end you will have to pixel peep to see a difference - so I don't think a picker helps there. A slider should be interesting for finetuning.

**Comment by ilia3101** [comment]
Have there been any multiplier changes since I opened this?

My 5D3 shots are looking awfully badly debayered :(

**Comment by masc4ii** [comment]
No idea... I don't think there is much happened. Maybe the wb / wbundo function is newer than that?!

**Comment by ilia3101** [comment]
Ok I think it is because I'm using 12 bit in crop_rec, so the image is 14 bit and only using a small 12 bit sized range (so amaze cant work very well)
